applications/ConvectionDiffusionApplication/python_scripts/bfecc_convection_diffusion_solver.py

```diff
@@ -1,133 +0,0 @@
-from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
-# importing the Kratos Library
-from KratosMultiphysics import *
-from KratosMultiphysics.ConvectionDiffusionApplication import *
```

Comment:
is there a BFECC solver which is working?

if not this one which?

Reply:
There is no BFECC solver working... In any case I think it shouldn't be difficult to integrate the BFECC convection in the current solvers.

Reply:
Resolving as the semiimplicit solver has been already merged.